lib/generators/setup_generator.rb

```diff
@@ -232,4 +263,11 @@ def stylesheet
     return "Replaced app's stylesheet assets with Orchid's.\nCreated global/ directory and file with app name for app-wide styling"
   end
 
+  # this method adapted from example at
+  # https://apidock.com/rails/Rails/Generator/Commands/Base/gsub_file
+  def sub_file(filepath, to_replace, replace_with)
+    content = File.read(filepath).sub(to_replace, replace_with)
+    File.open(filepath, 'wb') { |file| file.write(content) }
+  end
```

Comment:
Hehe I did look at that, but I preferred this older version because it's so much shorter than the newest version and I didn't think we needed any bells and whistles. Also I'll be honest I didn't understand all of their code at a glance, like
`say_status :gsub, relative_to_original_destination_root(path), config.fetch(:verbose, true)`
vs the version I did select, since it was pretty straightforward.

We can haggle about it, if you think we need the bells and whistles and not a straight up read file, change file, write to file approach like above